releng/release-tool/release-tool.go

```diff
@@ -308,7 +308,21 @@ func (r *releaseData) makeTag(branch string) error {
 
 	r.generateReleaseNotes()
 
-	_, err := gitCommand("-C", r.repoDir, "tag", "-s", r.tag, "-F", r.releaseNotesFile)
+	if !r.dryRun {
+		_, err := gitCommand("-C", r.repoDir, "add", r.releaseNotesFile)
+		if err != nil {
+			return err
+		}
+	}
+
+	if !r.dryRun {
+		_, err := gitCommand("-C", r.repoDir, "commit", "-m", fmt.Sprintf("CHANGELOG: Add release notes for %s", r.tag))
+		if err != nil {
+			return err
+		}
+	}
+
+	_, err := gitCommand("-C", r.repoDir, "tag", "-s", r.tag)
```

Comment:
`err` is unchecked?

Reply:
Pre-existing, but I agree that it's not idea. I'll add the check.